### PR TITLE
Updating cli docs for octopus.server.exe

### PR DESCRIPTION
**Pull request by michaelnoonan**
An automated update of the command line docs for octopus.server.exe.

Created on machine 'MICHAELNOON1449' by 'michaelnoonan'

### CODE REVIEW

File: docs/api-and-integration/octopus.server.exe-command-line/license.md [hunk 1]
```diff
@@ -14,9 +14,6 @@ Where [<options>] is any of:
 
       --instance=VALUE       Name of the instance to use
       --licenseBase64=VALUE  Base64 encoded version of the license key XML
-      --licenseFile=VALUE    Path to the file containing the license key XML
```
Comment (matt-richardson) [inline docs/api-and-integration/octopus.server.exe-command-line/license.md]: Pretty sure this shouldn't be removed

File: docs/api-and-integration/octopus.server.exe-command-line/new-certificate.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Where [<options>] is any of:
 
       --instance=VALUE       Name of the instance to use
       --replace              Replaces the existing certificate that Octopus
-                               Server uses to authenticate itself with its
```
Comment (matt-richardson) [inline docs/api-and-integration/octopus.server.exe-command-line/new-certificate.md]: This looks like a regression

File: docs/api-and-integration/octopus.server.exe-command-line/node.md [hunk 3]
```diff
@@ -17,13 +17,6 @@ Where [<options>] is any of:
                                a time
       --drain=VALUE          Set to true to prevent the node from executing
                                more tasks
-      --wait=VALUE           Seconds to wait for node to drain tasks. This
```
Comment (matt-richardson) [inline docs/api-and-integration/octopus.server.exe-command-line/node.md]: This should not be removed

**Comment by michaelnoonan** [comment]
Sorry, testing.

**Pull request closed without merging**